src/Interpreters/DDLTask.cpp

```diff
@@ -103,7 +103,10 @@ String DDLLogEntry::toString() const
     }
 
     if (version >= OPENTELEMETRY_ENABLED_VERSION)
-        wb << "tracing: " << this->tracing_context;
+        wb << "tracing: " << tracing_context;
```

Comment:
What's tracing_context btw?
If it's a string, we have a bug.

Reply:
Good point, here is a fix - #50045